app/views/layouts/_header.html.erb

```diff
@@ -90,4 +90,20 @@
   if (window.location.href.includes('search')) {
       searchBar.style.display = "block";
   }
+
+    /* for retaining select tag search query after a search*/
+  if(window.location.pathname==="/search"){
```

Comment:
This seems more like a workaround than a fix, ideally properly configured form should be handling this scenario itself

Reply:
Okay, I'll make the changes accordingly and make another pull request. Thanks for the feedback.